### PR TITLE
Java problem 24 , test file updated

### LINKED ISSUE #60: Add "Find Common Characters" Java Problem
## Issue Description:
This proposes adding the Find Common Characters problem to our Java code repository.

## Problem Statement
You can find the problem statement [here](https://leetcode.com/problems/find-common-characters/)

## Implementation
Please read [Contributing to Java Problems](https://github.com/SST-OSS/club-onboarding/tree/main/javaProblems#adding-a-new-problem) to get started with how to create new java problem and test for the same.

## Note
For this problem you have to create a new folder with name problem24 inside folder [javaProblems](https://github.com/SST-OSS/club-onboarding/tree/main/javaProblems)

### PR DESCRIPTION
**Pull request by krishna-p060**
test file updated according to requirement
fixes #60

### CODE REVIEW

**Comment by notpritam** [comment]
Please follow the guidelines of Readme correctly, you have not provided the example input and output

**Comment by probablyArth** [comment]
the readme still doesn't follow the one provided in the guide.
please go through it carefully

File: problem24/Solution.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Solution {
```
Comment (probablyArth) [inline problem24/Solution.java:1]: where is the solve method??

File: problem24/Test.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import java.util.List;
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class SolutionTest {
```
Comment (probablyArth) [inline problem24/Test.java:6]: The class is supposed to be named `Test`

**Review by probablyArth — CHANGES_REQUESTED** [review]

**Comment by probablyArth** [comment]
also please mention the issue # this solves

**Comment by AbhinavGupta-de** [comment]
@probablyArth LGTM!

**Review by probablyArth — APPROVED** [review]

**Pull request merged**